pkg/doc/verifiable/presentation.go

```diff
@@ -313,6 +328,57 @@ func NewPresentation(vpData []byte, opts ...PresentationOpt) (*Presentation, err
 	return vp, nil
 }
 
+// decodeCredentials decodes credential(s) embedded into presentation.
+// It could be defined as:
+// 1) string - it could be credential decoded into e.g. JWS.
+// 2) the same as 1) but as array - e.g. zero ore more JWS
+// 3) struct (should be map[string]interface{}) representing credential data model
+// 4) the same as 3) but as array - i.e. zero or more credentials structs.
```

Comment:
These are listing different possible formats for rawCred right? If so, perhaps you can make it explictly say "rawCred must be one of the following..."